python/guide/module/gluon/custom-layer.md

```diff
@@ -83,7 +85,7 @@ dense.initialize()
 dense(nd.random.uniform(shape=(2, 5)))
 ```
 
-We can also construct models using custom layers. Once we have that we can use it just like the built-in dense layer. The only exception is that in our case size inference is not automagic. Please consult the [MXNet documentation](http://www.mxnet.io) for details on how to do this.
+We can also construct models using custom layers. Once we have that we can use it just like the built-in dense layer. The only exception is that in our case size inference is not automatic.
```

Comment:
size inference is not automatic. would explain why not. also size -> shape

Reply:
done